libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -39,7 +39,11 @@ configured {es} output, specify the following minimal configuration:
 --
 [source, yml]
 --------------------
-xpack.monitoring.enabled: true
+xpack.monitoring:
```

Comment:
@ycombinator When I remove `xpack` from this, I no longer see Beats monitoring data in Kibana 7.0.

Reply:
Hi @dedemorton, that would be expected as the new `monitoring.*` settings were only introduced in 7.1.0: https://github.com/elastic/beats/pull/9260.

Just to avoid confusion, the `xpack.monitoring.*` settings will **continue to be available** in 7.1.0 and above. However they will be deprecated in favor of the `monitoring.*` settings.

Reply:
Ok, I need to backport this to 7.0, so I will use xpack.monitoring in this PR and then change it back after I backport to 7.1.